apps/block_scout_web/lib/block_scout_web/csp_header.ex

```diff
@@ -11,11 +11,11 @@ defmodule BlockScoutWeb.CSPHeader do
   def call(conn, _opts) do
     Controller.put_secure_browser_headers(conn, %{
       "content-security-policy" => "\
-        connect-src 'self' 'unsafe-inline' 'unsafe-eval' 'unsafe-hashes' #{websocket_endpoints(conn)} wss://*.bridge.walletconnect.org/ https://cdn.segment.com https://api.segment.io https://raw.githubusercontent.com/trustwallet/assets/ https://registry.walletconnect.org/data/wallets.json; \
+        connect-src 'self' 'unsafe-inline' 'unsafe-eval' 'unsafe-hashes' #{websocket_endpoints(conn)} wss://*.bridge.walletconnect.org/ https://cdn.segment.com https://api.segment.io https://raw.githubusercontent.com/trustwallet/assets/ https://registry.walletconnect.org/data/wallets.json https://*.google-analytics.com https://*.analytics.google.com https://*.googletagmanager.com;\
         default-src 'self'; \
-        script-src 'self' 'unsafe-inline' 'unsafe-eval' 'unsafe-hashes' https://www.google.com https://www.gstatic.com https://cdn.segment.com https://api.segment.io; \
+        script-src 'self' 'unsafe-inline' 'unsafe-eval' 'unsafe-hashes' https://www.google.com https://www.gstatic.com https://www.googletagmanager.com https://cdn.segment.com https://api.segment.io;\
```

Comment:
Segment stuff is removed elsewhere. Should this line, and line 14 still contain references to `https://api.segment.io`?